enhancements/two-nodes-openshift/2no.md

```diff
@@ -0,0 +1,510 @@
+
+Pacemaker - a Red Hat led [open-source project](https://clusterlabs.org/pacemaker/doc/) that works in conjunction with Corosync to provide general purpose fault tolerance and automatic failover for critical services and applications.
+
+Fencing - the process of “somehow” isolating or powering off malfunctioning or unresponsive nodes to prevent them from causing further harm, such as data corruption or the creation of divergent datasets.  
+
+Quorum - having the minimum number of members required for decision-making. The most common threshold is 1 plus half the total number of members, though more complicated algorithms predicated on fencing are also possible.
+ * C-quorum: quorum as determined by Corosync members and algorithms
+ * E-quorum: quorum as determined by etcd members and algorithms
+
+Split-brain - a scenario where a set of peers are separated into groups smaller than the quorum threshold AND peers decide to host services already running by other groups.  Typically results in data loss or corruption.   
+
+MCO - Machine Config Operator. This operator manages updates to node's systemd, cri-o/kubelet, kernel, NetworkManager, etc., and can write custom files to it, configurable by MachineConfig custom resources.
+
+ABI - Agent-Based Installer.
+
+## Summary
+
+Leverage traditional high-availability concepts and technologies to provide a container management solution suitable for customers with numerous geographically dispersed locations that has a minimal footprint but remains resilient to single node-level failures.
+
+## Motivation
+
+Customers with hundreds, or even tens-of-thousands, of geographically dispersed locations are asking for a container management solution that retains some level of resilience to node level failures, but does not come with a traditional three-node footprint and/or price tag.
+
+The need for some level of fault tolerance prevents the applicability of Single Node OpenShift (SNO), and a converged 3-node cluster is cost prohibitive at the scale of retail and telcos - even when the third node is a "cheap" one that doesn't run workloads.
+
+The benefits of the cloud-native approach to developing and deploying applications are increasingly being adopted in edge computing.
+This requires our solution to provide a management experience consistent with "normal" OpenShift deployments, and be compatible with the full ecosystem of Red Hat and partner workloads designed for OpenShift.
+
+### User Stories
+
+* As a large enterprise with multiple remote sites, I want a cost-effective OpenShift cluster solution so that I can manage containers without the overhead of a third node.
+* As a support engineer, I want a safe and automated method for handling the failure of a single node so that the downtime of the control-plane is minimized.
```

Comment:
We had stories around those, but removed them since the intention is that we should be able to host anything a 3-node control-plane can. 

Additionally, we moved workloads into a separate enhancement so this one can focus on the control-plane elements.